src/earn/EarnHome.test.tsx

```diff
@@ -159,16 +160,21 @@ describe('EarnHome', () => {
         <MockedNavigator
           component={EarnHome}
           params={{
-            activeEarnTab: EarnTabType.OpenPools,
+            activeEarnTab: EarnTabType.AllPools,
           }}
         />
       </Provider>
     )
 
+    // All Pools
     expect(
       queryByTestId('PoolCard/arbitrum-sepolia:0x460b97bd498e1157530aeb3086301d5225b91216')
-    ).toBeFalsy()
+    ).toBeTruthy()
+    expect(
+      getByTestId('PoolCard/arbitrum-sepolia:0x460b97bd498e1157530aeb3086301d5225b91216')
+    ).toBeTruthy()
```

Comment:
seems like these are asserting the same thing, should these be different?

Reply:
I think should be `PoolCard/ethereum-sepolia:0xe50fa9b3c56ffb159cb0fca61f5c9d750e8128c8`?

Reply:
Fixed in737593bcfa64e72ec22cea78c982fbbf3b03ee94!